source/extensions/omni.isaac.lab_tasks/omni/isaac/lab_tasks/manager_based/manipulation/reach/config/ur5e/__init__.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2022-2024, The Isaac Lab Project Developers.
+# All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+import gymnasium as gym
+
+from . import agents, joint_pos_env_cfg, joint_vel_env_cfg, ik_env_cfg
+
+##
+# Register Gym environments.
+##
+
+gym.register(
+    id="Isaac-Reach-UR5e-JointPos-v0",
```

Comment:
why do you give them `v0` suffixes? Is it still necessary if we have it version controlled?

Reply:
It is a convention in Isaac Gym to give versions as can be seen in Isaac Lab environments, I'll dig deeper to see if there is an actual meaning behind it.